components/menu/menu-item-link.js

```diff
@@ -13,6 +13,10 @@ class MenuItemLink extends MenuItemMixin(LitElement) {
 
 	static get properties() {
 		return {
+			/**
+			 * Name to use for download
+			 */
+			download: { type: String },
```

Comment:
Reading the spec... haha this is a weird attribute eh... it's a string and becomes the suggested filename if set to a string, but otherwise it's a boolean attribute and uses other mechanisms to determine the filename. Weird!

For the description in the comment, I'd suggest something like:
```suggestion
			 * Prompts the user to save the linked URL instead of navigating to it.
			 * Must be to a resource on the same origin.
			 * Can be used with or without a value, when set the value becomes the filename.
			 */
			download: { type: String },
```

Reply:
Along these lines - will also want to update the [readme for menu](https://github.com/BrightspaceUI/core/tree/master/components/menu#d2l-menu-item-link).